src/main/java/uk/gov/pay/api/resources/SearchDisputesResource.java

```diff
@@ -34,13 +44,42 @@ public SearchDisputesResource(SearchDisputesService searchDisputesService) {
+                            description = "Invalid parameters: from_date, to_date, from_settled_date, to_settled_date, status, display_size. See Public API documentation for the correct data formats",
+                            content = @Content(schema = @Schema(implementation = RequestError.class))),
+                    @ApiResponse(responseCode = "429", description = "Too many requests",
+                            content = @Content(schema = @Schema(implementation = ApiErrorResponse.class))),
+                    @ApiResponse(responseCode = "500", description = "Downstream system error",
+                            content = @Content(schema = @Schema(implementation = RequestError.class)))
+            }
+    )
+    public DisputesSearchResults searchDisputes(@Parameter(hidden = true)
+                                                    @Auth Account account,
+                                                @Parameter(description = "Returns disputes raised on or after the 'from_date' Date and time must be coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - 'YYYY-MM-DDTHH:MM:SSZ'.")
                                                 @QueryParam("from_date") String fromDate,
+                                                @Parameter(description = "Returns disputes raised before the 'to_date' Date and time must be coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - 'YYYY-MM-DDTHH:MM:SSZ'.")
                                                 @QueryParam("to_date") String toDate,
+                                                @Parameter(description = "Returns disputes settled on or after the 'from_settled_date'. Date must be in ISO 8601 format to date-level accuracy - 'YYYY-MM-DD'. Disputes are settled when your payment service provider takes the disputed amount from a payout to your bank account.<br><br>You can only use 'from_settled_date' to find payments raised on or after 8 August 2022.")
                                                 @QueryParam("from_settled_date") String fromSettledDate,
+                                                @Parameter(description = "Returns disputes settled before the 'to_settled_date'. 'Date must be in ISO 8601 format to date-level accuracy - 'YYYY-MM-DD'.<br><br>Disputes are settled when your payment service provider takes the disputed amount from a payout to your bank account. You can only use 'to_settled_date' to find payments raised on or after 8 August 2022.")
                                                 @QueryParam("to_settled_date") String toSettledDate,
+                                                @Parameter(description = "Returns disputes with a matching 'status'. 'status' reflects what stage of the dispute process a dispute is at.", example = "won",
+                                                        schema = @Schema(allowableValues = {"created", "needs_response", "under_review", "lost", "won"}))
```

Comment:
"created" isn't a valid external status for disputes. I think the request validation also needs to be updated to remove this.

Reply:
👍